setup.py

```diff
@@ -59,7 +59,7 @@ def _dbt_presto_version() -> str:
         ]
     },
     install_requires=[
-        "dbt-core==0.19.0",
-        "pyathena==2.1.1",
+        "dbt-core==0.19.1",
+        "pyathena==2.2.0",
```

Comment:
Upgrading to `pyathena` 2.2.0 will likely cause an issue within `connections.py`, specifically [L29](https://github.com/Tomme/dbt-athena/blob/70a91f667c42741b58aa650e47bdf848765df089/dbt/adapters/athena/connections.py#L29).

If you set `work_group` to default to `None` it should then be compatible.

This is due to the following PR [#221](https://github.com/laughingman7743/PyAthena/pull/221) within `pyathena`